content/docs/command-reference/remote/add.md

```diff
@@ -44,6 +30,25 @@ url = /tmp/dvc-storage
 remote = myremote
 ```
 
+`name` and `url` are required. The `name` is used to identify the remote and
+should be unique for the project.
+
+`url` specifies a location to store your data. It can represent a cloud storage
+service, an SSH server, network-attached storage, or even a directory in the
+local file system (see all the supported remote storage types in the examples
+below). Whenever possible, DVC will create a remote directory if it doesn't
```

Comment:
this is not true for all remotes (e.g. not for GDrive)

Reply:
The part about creating dirs? Yeah it is pre-existing text. I think that's what who ever wrote it meant by "Whenever possible" and "it won't create an S3 bucket though".

Should we remove this or add a full list of which types support this? Cc @efiop hi! Do you happen to know for which remote types this is the case? Thanks 🙂

Reply:
Bucket/Dir creation differs from remote to remote, so let's not mention it for now at all. We need to unify it in the code first.

Reply:
Removed.